wearipedia/devices/fitbit/fitbit_sense.py

```diff
@@ -0,0 +1,125 @@
+from datetime import datetime, time, timedelta
+
+from ...utils import bin_search, seed_everything
+from ..device import BaseDevice
+from .fitbit_authenticate import *
+from .fitbit_sense_fetch import *
+from .fitbit_sense_gen import *
+
+class_name = "Fitbit_sense"
+
+
+class Fitbit_sense(BaseDevice):
+    """This device allows you to work with data from the `Fitbit Sense <(https://www.fitbit.com/global/us/products/smartwatches/sense)>`_ device.
+    Available datatypes for this device are:
+
+    * `sleep`: sleep data
+    * `steps`: steps data
+    * `minutesVeryActive`: number of minutes with high activity
+    * `minutesLightlyActive`: number of minutes with light activity
+    * `minutesFairlyActive`: number of minutes with fair activity
+    * `distance`: in miles
+    * `minutesSedentary`: number of minutes with no activity
+    * `heart_rate_day`: heart rate data
+    * `hrv`: heart rate variability data
+    * `distance_day`: distance moved per day detailed by each minute
+
+    :param seed: random seed for synthetic data generation, defaults to 0
+    :type seed: int, optional
+    :param synthetic_start_date: start date for synthetic data generation, defaults to "2022-03-01"
+    :type synthetic_start_date: str, optional
+    :param synthetic_end_date: end date for synthetic data generation, defaults to "2022-06-17"
+    :type synthetic_end_date: str, optional
+    :param single_day: end date for real data data fetching, defaults to "2022-09-19"
```

Comment:
It looks like this class does not accept single_day as a parameter. If so, you can remove this part of the docstring